LWA/LWA_bifrost.py

```diff
@@ -1322,16 +1573,19 @@ def main():
         enable_thread_profiling()
 
     if not os.path.isdir(args.out_dir):
-        print('Output directory does not exist. Defaulting to current directory.')
-        args.out_dir = '.'
+        print("Output directory does not exist. Defaulting to current directory.")
+        args.out_dir = "."
 
     if args.removeautocorrs:
-        raise NotImplementedError('Removing autocorrelations is not yet properly'
-                                  + ' implemented.')
+        raise NotImplementedError(
```

Comment:
I have a new module aCorr in epic_optim branch that replaces the bfMap call for autocorrelation. It works fine so the "removeautocorrs" flag can be enabled again.

Reply:
Let's do that in a separate PR. We want to keep things that are functionally different in separate merges.

Reply:
Ah ok, I already tested the new code in py2.. Is it possible to test in py3 given that we don't have a functional py3 bifrost yet

Reply:
No, only py2 right now. But you're proposing changes to the code that aren't relevant to this PR. this is just changing syntax to be py3 compatible without changing any functionality. We should have a separate PR for your updates. That will make tracking the history easier.

Reply:
Ok, got it. I've removed those comments.